src/feature-selector-form/sub-components/selected-feature-details.tsx

```diff
@@ -22,16 +27,37 @@ export function SelectedFeatureDetails( { featureId }: Props ) {
 	};
 
 	let keywordsDisplay: ReactNode;
-	if ( keywords ) {
+	if ( keywords && keywords.length > 0 ) {
 		keywordsDisplay = <SortedKeywordList keywords={ keywords } />;
 	} else {
-		keywordsDisplay = <span className={ styles.noKeywords }>None</span>;
+		keywordsDisplay = (
+			<span data-testid={ 'keywords-no-result' } className={ styles.noResults }>
+				None
+			</span>
+		);
+	}
+
+	let repositoriesDisplay: ReactNode;
+	const dataTestId = 'selected-feature-repositories';
+
+	if ( repositories.length > 0 ) {
+		const repositoriesList = repositories.join( ', ' );
+		repositoriesDisplay = (
+			<span data-testid={ dataTestId } className={ styles.repositoriesList }>
+				{ repositoriesList }{ ' ' }
```

Comment:
**Nit (non-blocking)**: Did you mean for the extra space character at the end there? I know the linter can be really aggressive in adding those haha, so just wanted to check! 😄